nipype/algorithms/misc.py

```diff
@@ -35,6 +34,7 @@
                                BaseInterfaceInputSpec, isdefined,
                                DynamicTraitedSpec, Undefined)
 from ..utils.filemanip import fname_presuffix, split_filename
+from . import confounds
```

Comment:
This can be problematic. If you need some interface from confounds, please import that one interface only.